cmake/install-rules.cmake

```diff
@@ -1,3 +1,71 @@
 if(PROJECT_IS_TOP_LEVEL)
-  include(CPack)
+  set(
```

Comment:
Could you clarify what the changes in this file `install-rules.cmake` are needed for.

Reply:
I don't think the install command was doing anything previously as I hadn't populated this file from the cmake-init project template repo.

These changes install the contents of the `src` directory as well as any other library or archive files to an installation folder.

It also generates some additional cmake configuration and versioning files.

Reply:
Makes sense. Thanks for the clarification.